oresat_dxwifi/camera/camera.py

```diff
@@ -119,116 +63,144 @@ def __init__(
             )
         )
 
-    def check_seconds_per_video(self, duration, seconds_per_video):
+    def _check_seconds_per_video(self) -> bool:
+        """Returns whether seconds per video divides into the total duration"""
         is_valid = True
-        if duration % seconds_per_video != 0:
+        if self.total_duration % self.seconds_per_video != 0:
             is_valid = False
         return is_valid
 
-    # Make or overwrite the given directory
-    def fresh_dir(self, path):
+    @staticmethod
+    def _fresh_dir(path) -> None:
         # Overwrite
         if os.path.exists(path):
             for f in os.listdir(path):
                 p = os.path.join(path, f)
                 try:
                     shutil.rmtree(p)
-                    print("Removed directory: {}".format(f))
+                    logger.info("Removed directory: {}".format(f))
                 except OSError:
                     os.remove(p)
-                    print("Removed file: {}".format(f))
-            print("Cleaned Directory: {}".format(path))
+                    logger.info("Removed file: {}".format(f))
+            logger.info("Cleaned Directory: {}".format(path))
         else:
             # Make
             os.mkdir(path)
-            print("Created new directory: {}".format(path))
-
-    def overwrite_directories(self, frames_directory_path, video_directory_path):
-        # Make / overwrite directories
-        self.fresh_dir(frames_directory_path)
-        self.fresh_dir(video_directory_path)
-
-    def calculate_number_of_loops(self):
-        return self.total_duration // self.seconds_per_video
+            logger.info("Created new directory: {}".format(path))
 
-    def delete_frames(self):
-        # If requested, delete frames
+    def _delete_frames(self) -> None:
+        """If frame delete variable is true, deletes frames in image output directory"""
         if self.are_frames_deleted:
```

Comment:
Let's move the `if self.are_frames_deleted` to the caller, and let `_delete_frames()` just delete the frames